client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -47,8 +47,9 @@ const StockListComponent: FC = () => {
     ],
   });
   const navigate = useNavigate();
+  const filterWithInStock = { ...filterBy, hasPacksInStore: true };
```

Comment:
Why move this out? 🤔 You're passing in `hasPacksInStore` also in stocktake modal. Are you thinking there would be a case where you use the useStockList api for stock lines < 0? 🤔